AUTHORS

```diff
@@ -123,15 +135,18 @@ Koichiro Den                            den@klaipeden.com
 Kornilios Kourtis                       kornilios@isovalent.com
 Laurent Bernaille                       laurent.bernaille@datadoghq.com
 Lehner Florian                          dev@der-flo.net
+Lior Rozen                              liorrozen@users.noreply.github.com
```

Comment:
Maybe add a `.mailmap` entry in order to get the real e-mail, see 14e611d72ead11ae268725e021f9766ce7f0fa88:

```
Lior Rozen <liorr@tailorbrands.com>
```